create-base missing MAVEN_CENTRAL import

```shell
❯ bld create-base
Please enter a package name (for instance: com.example):
com.example
Please enter a project name (for instance: myapp):
BaseExample
Downloading finished successfully.
The project was successfully created at '/home/erik/dev/tmp/BaseExample'.
❯ cd BaseExample
❯ ./bld co
/home/erik/dev/tmp/BaseExample/src/bld/java/com/example/BaseExampleBuild.java:15: error: cannot find symbol
        repositories = List.of(MAVEN_CENTRAL);
                               ^
  symbol:   variable MAVEN_CENTRAL
  location: class com.example.BaseExampleBuild
```

--- a/src/main/resources/templates/bld/base/project_build.txt
+++ b/src/main/resources/templates/bld/base/project_build.txt
@@ -4,6 +4,8 @@ import rife.bld.BaseProject;
 
 import java.util.List;
 
+import static rife.bld.dependencies.Repository.MAVEN_CENTRAL;
+
 public class {{v projectBuild/}} extends BaseProject {
     public {{v projectBuild/}}() {
         pkg = "{{v package/}}";

